src/supporting-data/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,280 @@
+              providing insight into its potential role.
+            </p>
+            <p>
+              Experimental evidence or computational predictions supporting the
+              localization are often included.
+            </p>
+          </div>
+          {/* Literature Citations */}
+          <div className="uniprot-grid-cell--small-span-6 uniprot-grid-cell--medium-span-4">
+            <h3>
+              <Link
+                to={{
+                  pathname: LocationToPath[Location.CitationsResults],
+                  search: 'query=*',
+                }}
+              >
+                Literature Citations
+              </Link>
+            </h3>
+            <p>
+              Direct references to scientific studies supporting protein
+              annotations.
+            </p>
+            <p>
+              Includes experimental evidence such as enzymatic activities,
+              protein interactions, or structural data.
+            </p>
+          </div>
+          {/* Taxonomy */}
+          <div className="uniprot-grid-cell--small-span-6 uniprot-grid-cell--medium-span-4">
+            <h3>
+              <Link
+                to={{
+                  pathname: LocationToPath[Location.TaxonomyResults],
+                  search: 'query=*',
+                }}
+              >
+                Taxonomy
+              </Link>
+            </h3>
+            <p>
+              Classification of the protein based on the organism of origin,
+              linking to databases like NCBI Taxonomy.
+            </p>
+          </div>
+          {/* Cross-referenced Databases */}
+          <div className="uniprot-grid-cell--small-span-6 uniprot-grid-cell--medium-span-4">
+            <h3>
+              <Link
+                to={{
+                  pathname: LocationToPath[Location.DatabaseResults],
+                  search: 'query=*',
+                }}
+              >
+                Cross-referenced Databases
+              </Link>
+            </h3>
+            <p>
+              Links to databases like PDB (Protein Data Bank), Gene Ontology
+              (GO), Ensembl, and RefSeq.
+            </p>
+          </div>
+          {/* Human Diseases */}
+          <div className="uniprot-grid-cell--small-span-6 uniprot-grid-cell--medium-span-4">
+            <h3>
+              <Link
+                to={{
+                  pathname: LocationToPath[Location.DiseasesResults],
+                  search: 'query=*',
+                }}
+              >
+                Human Diseases
+              </Link>
+            </h3>
+            <p>
+              Provides insights into the relationship between proteins and human
+              health.
+            </p>
+            <p>
+              Includes disease annotations, pathogenesis information, variants
+              linked to diseases, and links to drug databases.
+            </p>
```

Comment:
With the spacing between paragraphs within a namespace description, it's a bit tricky to see where one namespace begins and ends. Perhaps just having single blocks of contiguous text for each namespace would help.